### ISSUE TITLE
How to open desktop application from Citrix environment

### ISSUE THREAD

**Issue by KiranBehara** [Microsoft/WinAppDriver#653]

**Comment by naeemakram** [comment]
You mean opening an app streaming from Citrix Zen App? That would be very helpful if sorted. I'll be watching this thread.

**Comment by PandaMagnus** [comment]
Yeah, I also would be excited to see if anyone cracked this. My team looked into it a couple years ago, and Citrix, as far as automation goes, was basically rendering up an image. We couldn't find anything useful to key off of. :( If anyone knows how to do this, it'd be great knowledge to share.

**Comment by KiranBehara** [comment]
> You mean opening an app streaming from Citrix Zen App? That would be very helpful if sorted. I'll be watching this thread.

Yes my scripts are running fine for the application in my desktop, but i need to run it from citrix and on servers

**Comment by KiranBehara** [comment]
> Yeah, I also would be excited to see if anyone cracked this. My team looked into it a couple years ago, and Citrix, as far as automation goes, was basically rendering up an image. We couldn't find anything useful to key off of. :( If anyone knows how to do this, it'd be great knowledge to share.

Can you share the details/research did by your team ?
